Content.Shared/ActionBlocker/ActionBlockerSystem.cs

```diff
@@ -109,10 +109,16 @@ public bool CanInteract(EntityUid user, EntityUid? target)
         /// </remarks>
         public bool CanUseHeldEntity(EntityUid user, EntityUid used)
         {
-            var ev = new UseAttemptEvent(user, used);
-            RaiseLocalEvent(user, ev);
+            var useEv = new UseAttemptEvent(user, used);
+            RaiseLocalEvent(user, useEv);
 
-            return !ev.Cancelled;
+            if (useEv.Cancelled)
+                return false;
+
+            var usedEv = new GettingUsedAttemptEvent(user);
+            RaiseLocalEvent(used, usedEv);
```

Comment:
I do think you may need two events for this, but this is confusing.

You have two options, I think?
1. Drop the targeted raise from UseAttemptEvent. We already know who the User is in UseAttemptEvent anyway.
2. Pass both user and used into GettingUsedAttemptEvent, rename it to be understandable, and fix the docstring on both the new event and this sending function.